### PR TITLE
feat: Update opentelemetry-instrumentation-confluent-kafka to support confluent-kafka <=2.7.0

### PR DESCRIPTION
**Pull request by rjduffner**
# Description

Update opentelemetry-instrumentation-confluent-kafka to support confluent-kafka <=2.7.0 

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# How Has This Been Tested?

- [X] Ran standard existing tests through tox

# Does This PR Require a Core Repo Change?
- [X] No.

# Checklist:

See [contributing.md](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/CONTRIBUTING.md) for styleguide, changelog guidelines, and more.

- [X] Followed the style guidelines of this project
- [x] Changelogs have been updated
- [x] Documentation has been updated

### CODE REVIEW

**Comment by rjduffner** [comment]
@xrmx, sorry to bother you, is there an @ available to get the owners of this instrumentation to look?

Thank you!

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added
+
+- `opentelemetry-instrumentation-confluent-kafka` Update opentelemetry-instrumentation-confluent-kafka to support confluent-kafka <=2.7.0
```
Comment (emdneto) [inline CHANGELOG.md:16]:
```suggestion
- `opentelemetry-instrumentation-confluent-kafka` Add support for confluent-kafka <=2.7.0
```

**Comment by emdneto** [comment]
> @xrmx, sorry to bother you, is there an @ available to get the owners of this instrumentation to look?
> 
> Thank you!

At this moment, I don't think we have someone looking at it. We welcome any member which has knowledge on the instrumentation to also help on reviews, so feel free to add yourself to componentes_owners.yml

**Review by emdneto — APPROVED** [review]

**Comment by rjduffner** [comment]
Thanks @emdneto and @lzchen!

**Comment by ashwinxbhc** [comment]
@rjduffner - When will the stable version from dev for 51b be released, because we have already upgraded out confluent kafka versions to 2.6.1 ?

**Pull request merged**